ios/Classes/SwiftLiveActivitiesPlugin.swift

```diff
@@ -133,21 +139,26 @@ public class SwiftLiveActivitiesPlugin: NSObject, FlutterPlugin, FlutterStreamHa
         attributes: liveDeliveryAttributes,
         contentState: initialContentState,
         pushType: nil)
+      if removeWhenAppIsKilled {
+        appLifecycleLifeActiviyIds.append(deliveryActivity.id)
+      }
       result(deliveryActivity.id)
     } catch (let error) {
       result(FlutterError(code: "LIVE_ACTIVITY_ERROR", message: "can't launch live activity", details: error.localizedDescription))
     }
   }
   
   @available(iOS 16.1, *)
-  func updateActivity(activityId: String, data: Dictionary<String, Any>, result: @escaping FlutterResult) {
+  func updateActivity(activityId: String, data: [String: Any], result: @escaping FlutterResult) {
     Task {
       for activity in Activity<LiveActivitiesAppAttributes>.activities {
-        if (activityId == activity.id) {
+        if activityId == activity.id {
           for item in data {
             sharedDefault!.set(item.value, forKey: item.key)
           }
+            
           
+          // Why is appGroupId empty here? Maybe add a comment.
           let updatedStatus = LiveActivitiesAppAttributes.LiveDeliveryData(appGroupId: "")
           await activity.update(using: updatedStatus)
```

Comment:
Can you maybe explain why the appGroupid is empty here?

Reply:
Seems to be an oversight...

Reply:
We need to remplace the empty string with ```self.appGroupId``` 👍

Reply:
Pushed the change 👍🏽